src/ui/svg/icons/venueTypes/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { ArtisticCourse } from './ArtisticCourse'
+import { Bookstore } from './Bookstore'
+import { ConcertHall } from './ConcertHall'
+import { CulturalCentre } from './CulturalCentre'
+import { Digital } from './Digital'
+import { Festival } from './Festival'
+import { Games } from './Games'
+import { Library } from './Library'
+import { Movie } from './Movie'
+import { Museum } from './Museum'
+import { MusicalInstrumentStore } from './MusicalInstrumentStore'
+import { Other } from './Other'
+import { PatrimonyTourism } from './PatrimonyTourism'
+import { PerformingArts } from './PerformingArts'
+import { RecordStore } from './RecordStore'
+import { ScientificCulture } from './ScientificCulture'
+import { VisualArt } from './VisualArt'
```

Comment:
tu peux faire ca aussi, afin d'éviter un import puis export : 
```suggestion
export { VisualArt as VisualArtIcon } from './VisualArt'
```